.gitignore

```diff
@@ -1,2 +1,2 @@
 PathFind
-PathFind.dSYM/*
```

Comment:
don't hardcode your binaries' names in `.gitignore`. Instead, compile to binaries with a file extension like `.out` and add `*.out` to your `.gitignore`.